MangoPay/PayOut.php

```diff
@@ -19,12 +33,24 @@ class PayOut extends Transaction
      */
     public $PaymentType;
 
+    /**
+     * One of PayOutPaymentDetails implementations, depending on $PaymentType
+     * @var object
+     */
+    public $PayoutPaymentDetails;
```

Comment:
What is the reason for adding this to the PayOut class? We already have it below.

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.